app/models/repair.rb

```diff
@@ -1,5 +1,9 @@
 class Repair < ApplicationRecord
   belongs_to :owner
+<<<<<<< HEAD
```

Comment:
OK, this file is in error.  The <<<< HEAD and ==== and >>>>> master show which lines were found to be in conflict during git conflict resolution.  You need to fix it to resolve the conflicts and save it again.  It won't run now, so I don't know how this build could work at all.